ads/google/imaVideo.js

```diff
@@ -1096,7 +1119,7 @@ export function zeroPad(input) {
 function onProgressClick(event) {
   // Call this logic once to make sure we still seek if the user just clicks
   // instead of clicking and dragging.
-  clearInterval(hideControlsTimeout);
+  clearTimeout(hideControlsTimeout);
```

Comment:
Some changes in this file to `clearTimeout` instead, since we're now using mocked timers and they would (rightly) fail when trying to `clearInterval` on an id created by `setTimeout`.